plugins/ipam/README.md

```diff
@@ -1,8 +1,8 @@
-# IPAM plugin
+# ECS IPAM plugin
 
 ## Overview
 
-The IPAM plugin will construct the IP, Gateway, Routes as a struct which will be
+The ECS IPAM plugin will construct the IP, Gateway, Routes as a struct which will be
```

Comment:
The ECS IPAM plugin constructs the IP, Gateway, and Routes, which are used as a struct